File: Data-Mining/assignment3/knn/mkdata.m
function [X, y] = mkdata

N1 = 200; N2 = 200; % Class sizes
X = [randn(2, N1), randn(2, N2) + 2];
y = [repmat(0, 1, N1), repmat(1, 1, N2)];

end

File: Data-Mining/assignment3/knn/mkdata.py
import numpy as np


def mkdata():
    n1 = 200
    n2 = 200  # Class sizes
    x = np.vstack((np.random.randn(n1, 2), np.random.randn(n2, 2) + 2))
    y = np.hstack((np.array([0] * n1), np.array([1] * n2)))
    return x, y

